### PR TITLE
Music releases

### PR DESCRIPTION
**Pull request by elinaarenstrup**
Link to my deployed project: https://thirsty-borg-dfa96a.netlify.com/

Our reflections on how this week's project turned out: It went quite smooth, had some trouble with the icons but we solved it by the help of our team members.

Kajsa Bootin & Elin Aarenstrup

### CODE REVIEW

File: code/src/Album.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./album.css";
+import "./icons.css";
+
+export const Album = props => {
+  return (
+    <section className="album-card">
+      <section className="album-cover">
+        <img className="album-img" src={props.image} alt="Album" />
```
Comment (ericaglimsholt) [inline code/src/Album.js]: Suggestion: add the album title to the `alt`

File: code/src/App.js [hunk 2]
```diff
@@ -1,12 +1,32 @@
-import React from 'react'
-import data from './data.json'
+import React from "react";
+import data from "./data.json";
+import { Album } from "./Album";
+import { Header } from "./Header";
 
-console.log(data)
+console.log(data);
```
Comment (ericaglimsholt) [inline code/src/App.js]:
```suggestion
```

File: code/src/App.js [hunk 3]
```diff
@@ -1,12 +1,32 @@
-import React from 'react'
-import data from './data.json'
+import React from "react";
+import data from "./data.json";
+import { Album } from "./Album";
+import { Header } from "./Header";
```
Comment (ericaglimsholt) [inline code/src/App.js]: You don't need the brackets when you are importing only one thing
```suggestion
import Album from "./Album";
import Header from "./Header";
```

**Comment by elinaarenstrup** [comment]
Eric, I can't find where i should add Kajsa Bootin to the PR. Could you help out?

**Pull request closed without merging**